### PR TITLE
Tag PlotlyJS.jl v0.6.3

### PR DESCRIPTION
**Pull request by attobot**
Repository: [sglyon/PlotlyJS.jl](https://github.com/sglyon/PlotlyJS.jl)
Release: [v0.6.3](https://github.com/sglyon/PlotlyJS.jl/releases/tag/v0.6.3)
Travis: [Travis Build Status](https://travis-ci.org/sglyon/PlotlyJS.jl/builds/247267185)
Diff: [vs v0.6.2](https://github.com/sglyon/PlotlyJS.jl/compare/5330d71a9757bdd371f36274c7fc382c085f3f06...7ed29185389f769889f4229a7aa83329195e08c1)
`requires` vs v0.6.2: 
```diff
--- 0.6.2/requires
+++ 0.6.3/requires
@@ -2,7 +2,7 @@
 JSON 0.7
 Blink 0.3.3
 Colors
-Compat 0.8.4
+Compat 0.25.2
 LaTeXStrings
 DocStringExtensions
 Juno
```
cc: @sglyon

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: PlotlyJS/versions/0.6.3/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.5
+JSON 0.7
+Blink 0.3.3
+Colors
+Compat 0.25.2
```
Comment (tkelman) [inline PlotlyJS/versions/0.6.3/requires]: Compat.Iterators was 0.18, doesn't look like you're using anything newer than that

Comment (sglyon) [inline PlotlyJS/versions/0.6.3/requires]: I think I need a newer compat for `@compat const X = Y` style typalias, right?

Comment (tkelman) [inline PlotlyJS/versions/0.6.3/requires]: that's also 0.18, but you only need the compat if there's a lhs type parameter

Comment (sglyon) [inline PlotlyJS/versions/0.6.3/requires]: got it, thanks. I've changed this now

**Pull request merged**